### PR TITLE
4086 add custom properties to cold chain equipment csv import

### LINKED ISSUE #4086: GAPS: Populate Asset custom properties via spreadsheet import
## Is your feature request related to a problem? Please describe 👀

#3868 added functionality to store supplementary custom data against Assets. 

The central user needs to be able to populate this data for all Assets via a spreadsheet import. 

| User story  | B.2.16 |
| :------------- | :------------- |
As a… | National EPI manager
I want… | to be able to bulk upload data for facilities from Excel
So that… | I don't need to update each facility manually

**Solution B.2.16**
The system will permit a bulk import of facility data to aid the central users in the scenario that the system is not distributed and therefore cannot leverage facility users to enter their own data.

### Moneyworks Jobcode 🧰

CCEIDEV

### PR DESCRIPTION
**Pull request by fergie-nz**
Fixes #4086 

# 👩🏻‍💻 What does this PR do?

Adds bulk import of custom fields to cold chain equipment

## 💌 Any notes for the reviewer?

You can try this out by importing new asset items.
Example csv should 

# 🧪 Testing

- [ ]  Go to Cold Chain -> Equipment
- [ ] Click import
- [ ] See additional custom property fields for asset import
- [ ] Try adding asset property fields on bulk import
- [ ] See bulk imported custom fields in database

# 📃 Documentation

- [ ] **Part of an epic**: documentation will be completed for the feature as a whole
- [ ] **No documentation required**: no user facing changes or a bug fix which isn't a change in behaviour
- [ ] **These areas should be updated or checked**: 
  1.
  2.

### CODE REVIEW

**Review by mark-prins — CHANGES_REQUESTED** [review]
🎉  very nice! it's working well Ferg - just putting in a suggestion which I've added to [this branch](https://github.com/msupply-foundation/open-msupply/compare/4086-add-custom-properties-to-cold-chain-equipment-csv-import...4086-add-custom-properties-to-cold-chain-equipment-csv-import(suggestion))

I think it's nicer, but see what you think

File: client/packages/coldchain/src/Equipment/ImportAsset/UploadTab.tsx [hunk 1]
```diff
@@ -25,6 +25,10 @@ import {
   AssetCatalogueItemFragment,
   useStore,
 } from '@openmsupply-client/system';
+import {
+  AssetPropertyFragment,
+  useAssetData,
+} from 'packages/system/src/Asset/api';
```
[the remark this answers is not in the document]
Comment (fergie-nz) [reply client/packages/coldchain/src/Equipment/ImportAsset/UploadTab.tsx:31]: Apologies Mark I think you've asked me the same thing 20 times now!

**Comment by fergie-nz** [comment]
> 🎉 very nice! it's working well Ferg - just putting in a suggestion which I've added to [this branch](https://github.com/msupply-foundation/open-msupply/compare/4086-add-custom-properties-to-cold-chain-equipment-csv-import...4086-add-custom-properties-to-cold-chain-equipment-csv-import(suggestion))
> 
> I think it's nicer, but see what you think

Nice Mark! Yes looks great, I like moving generic function into utils file

**Review by mark-prins — APPROVED** [review]
Cool - thanks for changes Ferg! 
let's 🚢

**Pull request merged**